controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/CloudTrialExpirer.java

```diff
@@ -160,6 +150,41 @@ private boolean notifyTenants() {
         }
     }
 
+    private void notifySignup(Tenant tenant) {
+        var consoleMsg = "Welcome to Vespa Cloud trial! [Manage plan](%s)".formatted(billingUrl(tenant));
+        queueNotification(tenant, consoleMsg, "Welcome to Vespa Cloud",
+                          "Welcome to Vespa Cloud! We hope you will enjoy your trial. " +
+                                  "Please reach out to us if you have any questions or feedback.");
+    }
+
+    private void notifyMidCheckIn(Tenant tenant) {
+        var consoleMsg = "You're halfway through the *14 day* trial period. [Manage plan](%s)".formatted(billingUrl(tenant));
```

Comment:
Single `*` is emphasis, not sure if you meant that, usually this would be bold (`**`)?